src/shared/common/search-handler.tsx

```diff
@@ -0,0 +1,66 @@
+import { get } from "dottie";
+import { History } from "history";
+import { publicKeyToAddress } from "iotex-antenna/lib/crypto/crypto";
+import { WithApolloClient } from "react-apollo";
+import { GetBlockMetasRequest } from "../../api-gateway/resolvers/antenna-types";
+import { GET_BLOCK_METAS } from "../queries";
+
+type Props = {
+  history: History;
+  client: WithApolloClient<{}>["client"];
+};
+
+export const handleSearch = async (
+  props: Props,
+  query: string
+): Promise<void> => {
+  const { history, client } = props;
+  const value = query.trim();
+
+  if (value.startsWith("io")) {
```

Comment:
You can use the regular expression as below for more accuracy:
```suggestion
  if (value.match(/^io[a-z0-9]{39}/)) {
```